src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx

```diff
@@ -862,9 +859,9 @@ export default class WorkloadPodLogs extends React.Component<WorkloadPodLogsProp
     });
   };
 
-  private entryToString = (entries: LogEntry[]): string => {
+  private entriesToString = (entries: LogEntry[]): string => {
     return entries.map(le => (this.state.showTimestamps ? `${le.timestamp} ${le.message}` : le.message)).join('\n');
   };
 
-  private hasEntries = (entries: LogEntry[]): boolean => entries.length > 0;
+  private hasEntries = (entries: LogEntry[]): boolean => !!entries && entries.length > 0;
```

Comment:
No need for the `||`, unless you set `entries` to be `LogEntry[] || undefined`.

Reply:
I originally did not have the truthy test and for whateve reason this code was being reached with entries as undefined.  It's possible other refactors fixed that issue, but leaving as it hurts nothing, except your eyes :)